src/components/input/input.stories.mdx

```diff
@@ -127,7 +123,45 @@ If the `rows` attribute is set, it will be used as the minimum height.
   </Story>
 </Canvas>
 
-### JavaScript Instructions
+You can also disable the `<textarea>` resizing ability by adding the `c-input--noResize` modifier:
+
+<Canvas>
+  <Story
+    name="Elastic Textarea with No Resize"
+    height="250px"
+    argTypes={{
+      rows: { type: { name: 'number' } },
+      class: { type: { name: 'string' } },
+    }}
+    args={{
+      ...elasticTextAreaConfig,
+      id: 'demo-elastic-no-resize',
+      class: 'c-input--elastic c-input--noResize js-elastic-textarea',
```

Comment:
Would it be simpler to apply this style to the existing `--elastic` modifier?

Reply:
Ooh, or maybe even restrict it to when a `rows` attribute is specified?

```css
.c-input--elastic[rows] {
  resize: none;
}
```

Reply:
In fact, I wonder if the modifier is necessary at all?

```css
.c-input[rows] {
  height: auto;
  resize: none;
}
```

Reply:
I like where you’re going with this, @tylersticka. I’ll make that change, thank you!

Reply:
@tylersticka I ended up adding [`resize: none` to the `c-input--elastic` modifier](https://github.com/cloudfour/cloudfour.com-patterns/pull/2098/commits/e557648aea4693370bf08815745d6b9e560df37b) based on the following assumptions:
- All Elastic TextAreas should have resizing disabled
- Not all Elastic TextAreas will have a `rows` attribute

Reply:
That makes sense. I generally wouldn't want to disable resizing on a non-elastic textarea.

Reply:
@gerardo-rodriguez I'd like to push back toward my original suggestion for a few reasons:

- As-is, our textareas _ignore_ the `rows` attribute. Try adding `rows="5"` to one of them, it won't work. I consider this a bug. Adding `height: auto` to `[rows]` would fix this.
- Our elastic textareas work by setting `rows`. Can you explain when an elastic textarea wouldn't have that attribute?
- If we disable resizing `c-input--elastic`, we're removing fallback behavior if the JavaScript is never applied. Whereas if we only apply this change when `[rows]` are set, it's basically progressive enhancement.

Reply:
One thought: Maybe instead of a modifier this should be a _state_ class, like `is-elastic`? That way it can truly be progressively enhanced.

Reply:
> As-is, our textareas ignore the rows attribute. Try adding rows="5" to one of them, it won't work. I consider this a bug. Adding height: auto to [rows] would fix this.

@tylersticka Excellent point!

---

> Our elastic textareas work by setting rows. Can you explain when an elastic textarea wouldn't have that attribute?

My mistake, I was basing my understanding on the [documentation](https://cloudfour-patterns.netlify.app/?path=/docs/components-input--elastic-textarea), which made `rows` sound optional to me:
> If the `rows` attribute is set, it will be used as the minimum height.

Scanning the JS, if `rows` is not set, a default of `2` is used.

---

> One thought: Maybe instead of a modifier this should be a state class, like `is-elastic`? That way it can truly be progressively enhanced.

Great idea, thank you!